src/strategies/Primitives/LimitSwapExactInput.ts

```diff
@@ -0,0 +1,115 @@
+import { TokenArgs, PrimitiveParamType, BigIntish } from '@brinkninja/types'
+import Token from '../Token'
+import InputTokenPrimitive from './InputTokenPrimitive'
+
+export type LimitSwapExactInputArgs = {
+  priceCurveAddress: string
+  priceCurveParams: string
+  signer: string
+  tokenIn: TokenArgs
+  tokenOut: TokenArgs
+  tokenInAmount: BigIntish,
+  fillStateParams: FillStateParamsJSON
+}
+
+export type FillStateParamsJSON = {
```

Comment:
should use the `FillStateParamsArgs` type instead, it's exported in `@brinkninja/types`